src/lldp_syncd/daemon.py

```diff
@@ -43,9 +43,18 @@ def parse_time(time_str):
             return sage;
     }
     :return: parsed age in time ticks (or seconds)
+             0 in case passed in string has negative time
+             due to setting system clock backwards
     """
+    if '-' in time_str:
+        return 0 
+
     days, hour_min_secs = re.split(LLDPD_UPTIME_RE_SPLIT_PATTERN, time_str)
-    struct_time = time.strptime(hour_min_secs, LLDPD_TIME_FORMAT)
+    try:
+        struct_time = time.strptime(hour_min_secs, LLDPD_TIME_FORMAT)
+    except ValueError:
+        logger.warning("Incorrect time {}, did you change system time?".format(hour_min_secs))
```

Comment:
I'd change the message to "Invalid time '{}'"

Reply:
Sure, but I would like to to keep the possible cause of system time modification as that will be  a pretty good hint to the debugger. Please suggest

Reply:
But change "did you change system time?", to "possibly system time changed?"